src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -77,8 +77,8 @@ public void storeMessage(Message message) {
   /**
```

Comment:
fyi the Datastore.java and Place.java are pretty different now. might be a good idea to pull (super sorry about this)